lib/kbsecret/cli/kbsecret-env

```diff
@@ -37,12 +37,11 @@ selected_records = if cmd.opts.all?
                      end
                    end
 
-selected_records.each do |record|
-  if cmd.opts.value_only?
-    puts record.value
-  elsif cmd.opts.no_export?
-    puts record.to_assignment
-  else
-    puts record.to_export
-  end
-end
+env_list = if cmd.opts.no_export?
```

Comment:
`env_list` might be a little misleading, since the return is a string and not a list. Maybe `output` or `env_output`?

Reply:
Fair point. `output` was too vague for me and I wanted to convey that it was a list of key:val pairs. `env_output` sounds good.

Reply:
Fixed!